docs/lib/predictions/fragments/android/identify-entity.md

```diff
@@ -49,19 +49,15 @@ public void detectEntities(Bitmap image) {
 <amplify-block name="Kotlin">
 
 ```kotlin
-fun detectEntities(image: Bitmap) {
-    Amplify.Predictions.identify(
-        IdentifyActionType.DETECT_ENTITIES,
-        image,
-        Consumer { result ->
-            val identifyResult = result as IdentifyEntitiesResult
-            val metadata = identifyResult.getEntities()[0]
-            Log.i("MyAmplifyApp", metadata.getBox()!!.toShortString())
-        },
-        Consumer { error ->
-            Log.e("MyAmplifyApp", "Entity detection failed", error)
-        }
-    )
+suspend fun detectEntities(image: Bitmap) {
+    try {
+        val result = Amplify.Predictions.identify(DETECT_ENTITIES, image)
+        val identifyResult = result as IdentifyEntitiesResult
+        val metadata = identifyResult.entities[0]
```

Comment:
Should we consider the index out or range possibility here? What about the `!!` below?

Reply:
I'll update to make it more defensive to the range/null-pointer issues.